server/aws-lsp-codewhisperer/src/language-server/securityScan/securityScanDiagnosticsProvider.ts

```diff
@@ -112,22 +114,16 @@ class SecurityScanDiagnosticsProvider {
         return true
     }
 
+    /**
+     * A default hover element for diagnostics is surfaced in the IDE
+     * that displays as `${diagnostic.Code}: ${diagnostic.Description}`
+     *
+     * Creating a custom lsp hover will add content to the IDE's existing hover,
+     * but will not replace the above default display text.
+     */
     handleHover = () => {
         this.lsp.onHover(({ position, textDocument }) => {
```

Comment:
Based on the comment, it seems like we do not need to handle this event then and want to let Visual Studio internally handle this. Can this handler be fully deleted?